packages/p2p-v2/src/pages/my-profile/screens/MyProfileCounterparties/MyProfileCounterpartiesHeader/MyProfileCounterpartiesHeader.scss

```diff
@@ -9,6 +9,10 @@
 
         .p2p-v2-search {
             max-width: 28rem;
+
+            @include mobile {
+                max-width: 100%;
```

Comment:
fix for search bar not full width on mobile:
<img width="456" alt="Screenshot 2024-01-30 at 3 03 47 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/14f2f5c0-71e5-40a8-9556-6caa0af96f39">